front/app/components/IdeasMap/index.tsx

```diff
@@ -318,7 +319,8 @@ const IdeasMap = memo<Props>(
     const onMapClick = useCallback(
       (event: any, mapView: MapView) => {
         // Save clicked location
-        setClickedMapLocation(esriPointToGeoJson(event.mapPoint));
+        const clickedPointProjected = projectPointToWebMercator(event.mapPoint);
+        setClickedMapLocation(esriPointToGeoJson(clickedPointProjected));
```

Comment:
Added a quick conversion here again of the event.mapPoint first into a "Web Mercator" projection so it works for the rest of the code.

Reply:
Here I'm also not sure what you mean with "so it works for the rest of the code".

Reply:
Sorry - I just meant the rest of the green changes in this file aren't anything new I added (they are the "rest of the code" I'm referring to here) :)

Sometimes I feel like it looks like there are lots of changes in a file, but I actually only added one line. I was trying to make that more clear, but I think I did the opposite :joy: